### ISSUE TITLE
iOs compiling for production error "expected argument type 'Date'"

### ISSUE THREAD

**Issue by defualt** [Leanplum/Leanplum-ReactNative-SDK#10]
"@leanplum/react-native-sdk": "1.0.12"

When compiling ios for TestFlight deployment: 

```
❌  /Users/asdf/node_modules/@leanplum/react-native-sdk/ios/LeanplumTypeUtils.swift:48:75: cannot convert value of type '() -> Date?' to expected argument type 'Date'

        messageDict["deliveryTimestamp"] = stringFromDate(leanplumMessage.deliveryTimestamp)
           ^



❌  /Users/asdf/node_modules/@leanplum/react-native-sdk/ios/LeanplumTypeUtils.swift:49:12: initializer for conditional binding must have Optional type, not '() -> Date?'

        if let expirationTimestamp = leanplumMessage.expirationTimestamp {
```

I was able to patch this with an npm `postinstall` script like this:

```
const problemFilePath = './node_modules/@leanplum/react-native-sdk/ios/LeanplumTypeUtils.swift';
const oldFileContent = fs.readFileSync(problemFilePath,'utf8');

fs.writeFileSync(
  problemFilePath,
  oldFileContent
    .replace(
      `if let expirationTimestamp = leanplumMessage.expirationTimestamp {`,
      `if let expirationTimestamp = leanplumMessage.expirationTimestamp() {`,
    )
    .replace(
      `messageDict["deliveryTimestamp"] = stringFromDate(leanplumMessage.deliveryTimestamp)`,
      `messageDict["deliveryTimestamp"] = stringFromDate(leanplumMessage.deliveryTimestamp())`),
  'utf8'
);
```

**Comment by e7mac** [comment]
Thanks for bringing this up. Please try [1.0.13](https://github.com/Leanplum/Leanplum-ReactNative-SDK/releases/tag/1.0.13) and let us know if that solves it for you.

**Comment by defualt** [comment]
Confirming 1.0.13 solves the issue.

**Comment by milos1290** [comment]
Closing as it was fixed in 1.0.13
